src/index.mjs

```diff
@@ -245,6 +256,8 @@ export function prefetch(urls, isPriority, checkAccessControlAllowOrigin, checkA
 * @return {Object} a Promise
 */
 export function prerender(urls, eagerness = 'immediate') {
+  urls = [].concat(urls);
+
```

Comment:
I wonder why this is necessary? Is it to create a copy to so that if the underlying object is modified during some async iteration to prevent unexpected behavior? I don't see this as being the case, since this is not an async function and below it just loops over the `urls` to add each to the `toPrerender` Set. 

I think this can just be removed.

```suggestion
```

Reply:
Hi @westonruter, this line was already present in the existing Speculation Rules implementation, so not really related to this PR.

It was done this way to ensure that URLs would be treated as array of strings.

@giorgiopellegrino to follow up in case an action is needed

Reply:
I see. The arg is `String | String[]`, so this does essentially cast to an array.